pkg/projects/Microsoft.NETCore.App/Microsoft.NETCore.App.depproj

```diff
@@ -65,6 +71,32 @@
     <Message Importance="low" Text="%(FilesToPackage.PackagePath) -> @(FilesToPackage)" />
   </Target>
 
+  <Target Name="GetDependenciesToPackage" Condition="'@(DependenciesToPackage)' != ''" DependsOnTargets="ResolveNuGetPackages" Returns="@(_DependenciesToPackageWithVersion)">
+    <!-- hack because current nuget task doesn't return version for ReferencedNuGetPackages,
```

Comment:
No issue is needed it will break when this happens.  We're not filing an issue to say "delete all project.jsons" right?